src/librustc_codegen_llvm/declare.rs

```diff
@@ -65,15 +65,24 @@ fn declare_raw_fn(
         }
     }
 
-    match cx.tcx.sess.opts.cg.opt_level.as_ref().map(String::as_ref) {
-        Some("s") => {
+    // FIXME(opt): this is kinda duplicated with similar code in attributes::from_fm_attrs…
```

Comment:
these are in the same crate; can we lift it out to a common method? Or is there some difference I'm missing...

Reply:
I extracted the common code into a function.